release-blocking-jobs.md

```diff
@@ -87,9 +87,31 @@ manual, human interpretation, we choose to move them to a separate dashboard.
 
 CI Signal Team members will regularly check the Informing Dashboard for new
 failures.  Unexplained failures that are not flakes should be considered reason
-to block the release until the failures are investigated and evaluated.  Unlike
-Blocking, however, jobs in Informing do not need to be green before authorizing
-a release, they can be failing for known and "acceptable" reasons.
+to block the release until the failures are investigated and evaluated.
+Unlike Blocking, however, jobs in Informing do not need to be green before
+authorizing a release, they can be failing for known and "acceptable" reasons.
+
+The criteria for deciding whether a failure in informing is to be ignored can be
+based on the following considerations:
+
+* Failure is due to a malfunction of the components that run a job and said
+  components are not related to Kubernetes
+  * For example, if a job is marked as failing because a test cluster could not be setup
+    but there are similar jobs that run the same tests, then the failing job
+    can be ignored.
+  * If a job stage stage, that is not related to the test results, fail
+    (e.g., couldn't upload some logs to a remote source) then the job can be
+    ignored.
+  * If a test-infra-related component fails and no there are no other factors
+    that could be responsible for the failure then the failure can be ignored.
+* Failure well understood and is caused by the configuration of a test and said
+  configuration is not related to an official SLO
+  * For exaple, if a test polls for metrics more fequently than these are
+    updated and fails because of it, then the failure may be ignored.
+* Job is flaking too frequently and is marked as failing
```

Comment:
I have a bit hard time understanding this section, could we maybe try to rephrase it a bit? Especially the first bullet point under this one. 

So are we saying that we can ignore jobs that are actually flaky but are already marked as failing because they flake too much? 🤔

Reply:
Thank you for catching this one!
I reworded it but essentially, what I wanted to communicate was a case as this one:
* we have a gce-cos-master-default job
* gce-cos-master-default job is marked as failing because the `overall` stage has failed in the last N runs
* `overall` stage is marked as failed for last N runs because there have been M unrelated e2e tests that flaked once or twice during the last N runs.
  * If there is a e2e test that flakes multiple times during the last N runs then we DONOT ignore

wdyt?